package.json

```diff
@@ -23,7 +23,7 @@
     "github-release": "github-release-from-changelog",
     "lint": "yarn lint:js . && yarn lint:ts && yarn lint:md .",
     "lint:js": "cross-env NODE_ENV=production eslint --cache --cache-location=.cache/eslint --ext .js,.jsx,.json",
-    "lint:ts": "tslint -p . -t stylish -e '{**/node_modules/**,**/dist/**,lib/cli/test/**,**/storybook-static/**}' **/*.ts",
+    "lint:ts": "tslint -p . -c tslint.json -t stylish **/*.ts",
```

Comment:
This will lint all the files on each commit. Please remove the `**/*.ts` part and add it in `lint` script instead (as it's done for eslint)